docs/index.rst

```diff
@@ -2,23 +2,26 @@ Flocker Documentation
 =====================
 
 Flocker is a data volume manager and multi-host Docker cluster management tool.
-With it you can control your data using the same tools you use for your stateless applications.
-This means that you can run your databases, queues and key-value stores in Docker and move them around as easily as the rest of your app.
+With it, you can control your data using the same tools you use for your stateless applications.
+This means that you can run your databases, queues and key-value stores in Docker and move them around, as easily as the rest of your app.
 
-With Flocker's command line tools and a simple configuration language, you can deploy your Docker-based applications onto one or more Linux hosts.
-Once deployed, your applications will have access to the volumes you have configured for them.
-Those volumes will follow your containers when you use Flocker to move them between different hosts in your Flocker cluster.
+With Flocker's command line tools and simple configuration language, you can deploy your Docker-based applications onto one or more Linux hosts.
+Once deployed, your applications will have access to the volumes configured for them.
+When you use Flocker to move containers between different hosts in your Flocker cluster, those volumes will follow.
 
 Contents:
 
 .. toctree::
    :maxdepth: 2
 
-   introduction
-   gettingstarted/index
-   indepth/index
-   advanced/index
-   gettinginvolved/index
-   roadmap/index
-   faq/index
-   authors
+   flocker/index
+   flocker/introduction/index
+   flocker/using/index
+   flocker/concepts/index
+   flocker/reference/index
+   flocker/releasenotes/index
+   flocker/faq/index
+   cluster/index
+   cluster/getinvolved/index
+   cluster/authors
+   
```

Comment:
New line at EOF (we should really use [doc8](https://pypi.python.org/pypi/doc8) or similar).